<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1149</width>
    <height>561</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>101</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="icons.qrc">
      <normaloff>:/Icons/to-do-list.png</normaloff>:/Icons/to-do-list.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>101</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="icons.qrc">
      <normaloff>:/Icons/books.png</normaloff>:/Icons/books.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>210</y>
      <width>101</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="icons.qrc">
      <normaloff>:/Icons/user-data.png</normaloff>:/Icons/user-data.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>300</y>
      <width>101</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="icons.qrc">
      <normaloff>:/Icons/settings.png</normaloff>:/Icons/settings.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>390</y>
      <width>101</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="icons.qrc">
      <normaloff>:/Icons/theme.png</normaloff>:/Icons/theme.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>0</y>
      <width>1001</width>
      <height>531</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>3</number>
    </property>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Managing</string>
     </attribute>
     <widget class="QTableWidget" name="tableWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>111</y>
        <width>951</width>
        <height>381</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>Book Code</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Title</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Author</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Category</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Price</string>
       </property>
      </column>
     </widget>
     <widget class="QPushButton" name="pushButton_6">
      <property name="geometry">
       <rect>
        <x>820</x>
        <y>40</y>
        <width>91</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>URW Bookman</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Confirm</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>241</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>URW Bookman</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="inputMask">
       <string/>
      </property>
      <property name="placeholderText">
       <string>Enter Title</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>50</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <item>
       <property name="text">
        <string>Returning</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Issuing</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>50</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>URW Bookman</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="lineWidth">
       <number>1</number>
      </property>
      <property name="text">
       <string>Type</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>50</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>URW Bookman</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Days Of Issue</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_2">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>50</y>
        <width>91</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <item>
       <property name="text">
        <string>1 Week</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2 Weeks</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>3 Weeks</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>4 Weeks</string>
       </property>
      </item>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Book</string>
     </attribute>
     <widget class="QTabWidget" name="tabWidget_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1001</width>
        <height>511</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab_5">
       <attribute name="title">
        <string>Add Book</string>
       </attribute>
       <widget class="QLineEdit" name="lineEdit_3">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>100</y>
          <width>301</width>
          <height>41</height>
         </rect>
        </property>
       </widget>
       <widget class="QPlainTextEdit" name="plainTextEdit">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>150</y>
          <width>451</width>
          <height>261</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_7">
        <property name="geometry">
         <rect>
          <x>500</x>
          <y>430</y>
          <width>151</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>URW Bookman</family>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Save</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>100</y>
          <width>121</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>URW Bookman</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Book Title</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_6">
       <attribute name="title">
        <string>Book Data</string>
       </attribute>
       <widget class="QPlainTextEdit" name="plainTextEdit_2">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>150</y>
          <width>451</width>
          <height>261</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_8">
        <property name="geometry">
         <rect>
          <x>620</x>
          <y>430</y>
          <width>151</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>URW Bookman</family>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Confirm</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_6">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>100</y>
          <width>301</width>
          <height>41</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_13">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>100</y>
          <width>121</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>URW Bookman</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Book Title</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_14">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>30</y>
          <width>131</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>URW Bookman</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Book Search</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_7">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>30</y>
          <width>301</width>
          <height>41</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_9">
        <property name="geometry">
         <rect>
          <x>660</x>
          <y>30</y>
          <width>181</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>URW Bookman</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Search</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_10">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>430</y>
          <width>151</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>URW Bookman</family>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Delete</string>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Users</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Tweaks</string>
     </attribute>
    </widget>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>150</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>URW Bookman</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Book Code</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>260</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>URW Bookman</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Author</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_17">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>310</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>URW Bookman</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Publisher</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_18">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>380</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>URW Bookman</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Book Price</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_9">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>270</y>
      <width>231</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_10">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>320</y>
      <width>231</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_9">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>380</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_10">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>150</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_19">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>210</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>URW Bookman</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Category</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_11">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>220</y>
      <width>231</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1149</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
